README.md

```diff
@@ -71,109 +113,88 @@ This script writes 5 files:
-```
+Common crawl offers to ways to download their corpus: via http and via Amazon's
+S3 buckets. The first version is getting less and less support and since 2023 it
+has been throttled down. So we switched over to use the S3 version.
 
-This step takes a while, so it make sense to [distribute the work among
-a cluster of machines](#tech). However, it is a bit more involved than
-distributing other scripts, as this one has been inherited from the old
-repository. The differences are:
+This requires a valid AWS account to be configured using ENV variables with the
+default names: AWS_ACCESS_KEY, AWS_SECRET_ACCES_KEY, etc.
 
-1. The script requires a `glob` (possibly wildcard) expression that expands to
-   a list of input files, and not an input directory. So the place of the host
-   name in the value to the `-i` argument must be specified manually (with
-   `{}`, see below)
-1. The number of processes must be set to 0. This eliminates the `-P` argument
-   from the Python command line. We need to do this because first, `-P` means
-   something else in `download_pages.py`, and second, the script uses threads
-   anyway.
+Thanks to Amazon's support for the Common Crawl project, downloading is free.
 
+Pages in the (filtered, deduplicated) index can be downloaded by the command
 ```
-ansible-playbook -i hosts distribute_files.yml -e
-    '{"input_dir": "/mnt/data/lang/Hungarian/cc_corpus/2019/cc_index_dedup",
-      "output_dir": "/mnt/data/lang/Hungarian/cc_corpus/2019"}'
-
-ansible-playbook -i hosts python.yml -e
-    '{"python_script": "download_pages.py",
-      "log_file": "2019_download.log",
-      "working_dir": "/mnt/data/lang/Hungarian/cc_corpus/",
-      "arguments": "-o $output_dir -e warc.gz -i $input_glob",
-      "per_host_args": {"input_glob": "\"2019/cc_index_dedup_{}/*.gz\"",
-                        "output_dir": "2019/cc_downloaded/"}, "processes": 0}'
+download_pages.py -i 03_index_dedup/2019-04 -o 04_downloaded/2019-04
+--index_output_dir 04a_index_sorted/2019-04 --error_file 04b_download_errors
 ```
 
+It is highly recommended to set the number of processes (-P) as high as
+possible in your environment. The reason for this is that S3 does not allow 
+the download of multiple slices from a single file during a single request.
+This means that the number of download requests we make will be comparable
+to the total number of documents we download.
+
 ### Remove boilerplate
 
-Boilerplate code is removed with `justext`, which also splits the data into
-paragraphs. The script to run is `remove_boilerplate.py`. Since we have already
-split up the data between hosts, we can distribute boilerplate removal as well.
+We have to turn webscraped pages into plain text representations. We also want
+to remove "boilerplate", that is text strings which are not parts of the 
+document itself. For example the texts of the menus of a webpage, the legal
+disclaimers, privacy and cookie policy explanations and such.
```

Comment:
and such -> etc (there is an and a few words prior)

Reply:
Ok.